vipps-report-api-faq.md

```diff
@@ -64,16 +76,33 @@ We aim to provide more information through this API in the future.
 for *Vippsnummer* sale units. The Report API can only be used to sale units
 that have API access (which *Vippsnummer* sale units do not have.)
 
+## Why are the users' names, transaction texts, etc. not available in the Report API?
+
+The Report API is primarily for accounting partners who will use the API to integrate
+with their accounting systems, allowing them to provide the accounting information to their merchants.
+
+The reports on
+[portal.vipps.no](https://portal.vipps.no)
+provide more details.
+The users' names, transaction texts and other information may be regulated by GDPR
+and therefore require consent from the merchant. This consent may be given on
+[portal.vipps.no](https://portal.vipps.no),
+but not when using the Report API.
+
+The Report API _may_ be extended to contain more information later, and this FAQ
+will be updated if there are any changes.
+There are no specific plans to do this yet.
+
 ## Is the Report API available for the test environment?
 
-Unfortunately not because the
+Nope. The
```

Comment:
See previous comment. I don't think "Nope" is good in customer-facing docs because it seems like we just really don't care about the users.